examples/github_search/pubspec.yaml

```diff
@@ -4,30 +4,30 @@ homepage: https://primeholding.com/
 version: 1.0.0+1
 
```

Comment:
https://dart.dev/guides/libraries/private-files it seems that for this app the `pubspec.lock` is in `.gitignore`, but the documentation says that for apps the file should be committed. I had one error because the locally generated file contained a different version for `file` dependency and I couldn't run the app. So I think that this file should be committed to the repo.